LICENSE

```diff
@@ -14,10 +14,27 @@ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 License for the specific language governing permissions and limitations
 under the License.
 
+All Linux kernel driver code included in "xocl" authored by Xilinx is
```

Comment:
is this "xocl" the name of the directory or the driver? if it's driver name, should we also mention xclmgmt?

Reply:
Thanks will update